utils/utils.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/go-openapi/runtime"
+)
+
+// ClientConfig contains configuration passed to the individual API Clients.
+type ClientConfig struct {
+	// NumRetries contains the number of attempted retries
+	NumRetries int
+	// RetryTimeout says how long to wait before retrying a request
+	RetryTimeout time.Duration
```

Comment:
@nikimanoledaki - Do we have a Global timeout of the request and any safeguards or docs to inform users to not exceed this limit?